modules/aws-outposts-prereq-storage-csi.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-prereq-storage-csi_{context}"]
+= Preparing the CSI driver to support AWS Outposts
+
+You must update the Container Storage Interface (CSI) driver default storage class to be compatible with AWS Outposts.
+To prevent the CSI Operator from recreating the initial default storage class, you must set its state to *Unmanaged*.
```

Comment:
To help out users, you can xref them to this topic for info about managing the default storage class and how to change this parameter: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/storage/using-container-storage-interface-csi#overview.

Reply:
Per our conversation: leaving as-is for now, with plan for snippet-based reuse of content in the respective modules. Leaving a to-do note in this module with the next steps to do that.

Thank you again for looking this over and for your advice/expertise! :)